Descent3/ctlconfig.cpp

```diff
@@ -882,14 +886,14 @@ void wpnsel_cfg_screen::realize() {
 
   m_reset_btn.Create(m_menu, UID_RESETDEFAULTS, TXT_RESETTODEFAULT, KEYCFG_EXTRAS_X + m_sheet->X(),
                      KEYCFG_EXTRAS_Y + m_sheet->Y(), NEWUI_BTNF_LONG);
-  m_primary_title.Create(m_menu, &UITextItem(MONITOR9_NEWUI_FONT, TXT_PRIMARY), WPNSEL_PRIMARY_X,
-                         WPNSEL_PRIMARY_Y - 28);
-  m_secondary_title.Create(m_menu, &UITextItem(MONITOR9_NEWUI_FONT, TXT_SECONDARY), WPNSEL_SECONDARY_X,
-                           WPNSEL_SECONDARY_Y - 28);
-  m_primary_low.Create(m_menu, &UITextItem(MONITOR9_NEWUI_FONT, TXT_LOWPRIORITY), WPNSEL_PRIMARY_X,
-                       WPNSEL_SECONDARY_Y - 11);
-  m_secondary_low.Create(m_menu, &UITextItem(MONITOR9_NEWUI_FONT, TXT_LOWPRIORITY), WPNSEL_SECONDARY_X,
-                         WPNSEL_SECONDARY_Y - 11);
+  auto item = UITextItem(MONITOR9_NEWUI_FONT, TXT_PRIMARY);
+  m_primary_title.Create(m_menu, &item, WPNSEL_PRIMARY_X, WPNSEL_PRIMARY_Y - 28);
+  auto item2 = UITextItem(MONITOR9_NEWUI_FONT, TXT_SECONDARY);
```

Comment:
could be named itemPrimaryTitle to make sure the function below is using the right one (which is not the case). The same goes in other places where multiple items are used